Library/Formula/rcssserver.rb

```diff
@@ -67,7 +67,31 @@ def install
 
     resources.each do |r|
       r.stage do
-        system "./bootstrap" if build.head?
+        Patch.create(:p1, <<EOD).apply
```

Comment:
> Also need a brief in-formula explanation of why the patch is necessary and where it has been reported.

I reported the patch to the following URL.
https://sourceforge.net/p/sserver/patches/10/

The upstream seems to work not so often.
I am afraid that it will take a long time...
People who would like to install it by blew may have the trouble and
waste their time.
I would like you to accept this commit .

> We don't currently support patching resources. We may need to wait for an upstream release.
My commited formula applies the patch.
The formula works well without the errors.
Is there anything wrong with the formula or the patch?

Is there any rules for writing formula?
Ex. you may not write a code like Patch.create(:p1, <<EOD).apply